packages/core/upload/server/services/folder.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+const uuid = require('uuid/v4');
+const { trimChars, trimCharsEnd, trimCharsStart } = require('lodash/fp');
+
+// TODO: to use once https://github.com/strapi/strapi/pull/12534 is merged
```

Comment:
The PR was merged already: could you resolve this TODO?

Reply:
Yes, I'll update ML-folder with master